.github/workflows/docker.yml

```diff
@@ -3,42 +3,72 @@ name: Docker CI
 on:
   push:
     branches:
-      - "main"
+      - '*'
```

Comment:
I'm wondering that why we don't run job on specific branches? The other branch seems to build for nothing in this case.

Reply:
It'll be trigger on every branch, but jobs are executed following the name of the branch. 
It's mainly to trigger the build without push every where and check your project can compile.  Because it's a public repo, ci time is free so no trouble here. In case of push on release for example, cache should avoid double build
 
  ```
   - name: Build
        if: ${{ startsWith(github.ref,'refs/heads/') }}
        uses: docker/build-push-action@v4
        with:
          file: ./Dockerfile
          push: false
          cache-from: type=gha
          cache-to: type=gha,mode=max
          build-args: ENV=prod
```

Reply:
It's just a little bit my environmental concern for building all branch. It could save some energy to protect our earth. I'd tend to build for all PRs and important branches. but it's unlikely that Multisig will have a lot of active bunch at the same time.